includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -844,6 +847,20 @@ private function get_section_export_defaults_main()
                     "woocommerce-myparcel"
                 ),
             ],
+            [
+                "name"      => WCMP_Settings::SETTING_RETURN_IN_THE_BOX,
+                "label"     => __("Print return label directly", "woocommerce-myparcel"),
+                "type"      => "select",
+                "options"   => [
+                    self::NOT_ACTIVE        => __("No", "woocommerce-myparcel"),
+                    self::NO_OPTIONS        => __("No options", "woocommerce-myparcel"),
```

Comment:
ik vind dit niet duidelijk. Misschien "Without options" en "Options equal to parent shipment" van maken.